content/cli/kes-ls/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: kes ls
+date: 2023-03-03
+lastmod: :git
+draft: false
+tableOfContents: true
+---
+
+{{< admonition title="Replaces deprecated commands" type="note" >}}
+The `kes ls` command replaces the following commands, which are deprecated:
+
+- `kes key ls`
+- `kes policy ls`
+- `kes identity ls`
+{{< /admonition >}}
+
+## Overview
+
+Returns a list of keys, policies, or identities.
+
+## Syntax
+
+```sh
+kes ls                         \
+    --api-key, -a <string>     \
+    --server, -s <HOST[:PORT]> \
+    --json                     \
+    --identity, -i             \
+    --policy, -p               \
+    --insecure, -k
```

Comment:
What is the default behavior? just keys?

Are the options mutually exclusive? E.g.

```

kes ls

kes ls --identity

kes ls --policy

kes ls --identity --policy
```

These would have different outputs - how would they be different? Might be an enginering question after testing locally.

Reply:
Clarified what happens with each of the above options. (The last isn't a valid command; it returns an error.)